loop/routes/rooms.js

```diff
@@ -428,6 +429,10 @@ module.exports = function (apiRouter, conf, logError, storage, auth,
               });
             });
         },
+        handleUpdateStatus: function(req, res) {
+          // Room status update is validated and logged in middlewares
```

Comment:
I would put the code that handle the status logging here rather than in a middleware that will be used only for this action.